adoc/admin-monitoring-stack.adoc

```diff
@@ -0,0 +1,840 @@
+= Monitoring Stack
+
+[IMPORTANT]
+====
+This is not an officially supported recommendation and does not claim complete coverage of any use case in a production environment.
+
+The described monitoring approach in this document is a generalized example of one way of monitoring a {productname} cluster.
+
+Please apply best practices to develop your own monitoring approach using the described examples and available health checking endpoints.
+====
+
+== Introduction
+
+This document aims to describe monitoring in a {kube} cluster.
+
+The monitoring stack consists of a metrics server, a visualization platform, and an ingress controller for authentication.
+
+* *Prometheus*
++
+Prometheus is an open-source metrics server with a dimensional data model, flexible query language, efficient time series database and modern alerting approach. The time series collection happens via a pull mode over HTTP.
++
+The Prometheus consists of multiple components:
++
+ ** Prometheus server: scrapes and stores data to time series database
+ ** https://prometheus.io/docs/alerting/alertmanager/[Alertmanager] handles client alerts, sanitizes duplicates and noise and routes them to configuratble receivers.
```

Comment:
Consistently use `Alertmanager` instead of `AlertManager`.

ref: https://prometheus.io/docs/alerting/alertmanager/